azurerm/internal/acceptance/data.go

```diff
@@ -82,3 +83,30 @@ func BuildTestData(t *testing.T, resourceType string, resourceLabel string) Test
 
 	return testData
 }
+
+func (td *TestData) RandomIntOfLength(len int) int {
```

Comment:
some resources (e.g. Storage/Windows VM's) require this to support 4-8 chars; so I think we could make this `MMYYR..`?